packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {
+  css,
+  BaseComponent
+} from '../../Utilities';
+import { IResizeGroupProps, IResizeGroup } from './ResizeGroup.Props';
+import styles = require('./ResizeGroup.scss');
+
+export interface IResizeGroupState {
+  renderedData?: any;
+  measuredData: any;
+  shouldMeasure?: boolean;
+}
+
+export class ResizeGroup extends BaseComponent<IResizeGroupProps, IResizeGroupState> implements IResizeGroup {
+
+  public static defaultProps = {
+    data: {}
+  };
+
+  private _root: HTMLElement;
+  private _measured: HTMLElement;
+
+  constructor(props: IResizeGroupProps) {
+    super(props);
+    this.state = {
+      shouldMeasure: true,
+      renderedData: null,
+      measuredData: { ...this.props.data },
```

Comment:
Since you're setting measuredData here you'll also need to set it in ` componentWillReceiveProps ` or you'll wind up with some weird scenarios where it might not appear to update properly.